models/src/it/java/io/sphere/sdk/orders/commands/OrderImportCommandTest.java

```diff
@@ -0,0 +1,356 @@
+        testOrderAspect(
+                builder -> builder.shippingAddress(address),
+                order -> assertThat(order.getShippingAddress()).isPresentAs(address)
+        );
+    }
+
+    @Test
+    public void taxedPrice() throws Exception {
+        final MonetaryAmount totalNet = EURO_10;
+        final double v = 0.19;
+        final MonetaryAmount totalGross = totalNet.multiply(v + 1);
+        final MonetaryAmount taxes = totalGross.negate().add(totalNet);
+        final TaxedPrice taxedPrice = TaxedPrice.of(totalNet, totalGross, asList(TaxPortion.of(v, taxes)));
+        testOrderAspect(
+                builder -> builder.taxedPrice(taxedPrice),
+                order -> assertThat(order.getTaxedPrice()).isPresentAs(taxedPrice)
+        );
+    }
+
+    @Test
+    public void customerEmail() throws Exception {
+        withCustomer(client(), customer ->
+                testOrderAspect(
+                        builder -> builder.customerEmail(customer.getEmail()),
+                        order -> assertThat(order.getCustomerEmail()).isPresentAs(customer.getEmail())
+                )
+        );
+    }
+
+    @Test
+    public void customerId() throws Exception {
+        withCustomer(client(), customer -> {
+            final String customerId = customer.getId();
+            testOrderAspect(builder -> {
+                        builder.customerId(customerId);
+                    },
+                    order -> assertThat(order.getCustomerId()).isPresentAs(customerId));
+        });
+    }
+
+    @Test
+    public void customerGroup() throws Exception {
+        withCustomerInGroup(client(), (customer, customerGroup) -> {
+            final String customerId = customer.getId();
+            testOrderAspect(builder -> builder.customerId(customerId).customerGroup(customerGroup),
+                    order -> assertThat(order.getCustomerGroup()).isPresentAs(customerGroup.toReference()));
+        });
+    }
+
+    @Test
+    public void orderNumber() throws Exception {
+        final String orderNumber = randomString();
+        testOrderAspect(builder -> builder.orderNumber(orderNumber),
+                order -> assertThat(order.getOrderNumber()).isPresentAs(orderNumber));
+    }
+
+    @Test
+    public void billingAddress() throws Exception {
+        final Address billingAddress = randomAddress();
+        testOrderAspect(builder -> builder.billingAddress(billingAddress),
+                order -> assertThat(order.getBillingAddress()).isPresentAs(billingAddress));
+    }
+
+    @Test
+    public void getCompletedAt() throws Exception {
+        final Instant completedAt = Instant.now().minusSeconds(5555);
+        testOrderAspect(builder -> builder.completedAt(completedAt),
+                order -> assertThat(order.getCompletedAt()).isPresentAs(completedAt));
+    }
+
+    private void testOrderAspect(final Consumer<ImportOrderBuilder> orderBuilderConsumer, final Consumer<Order> orderConsumer) {
```

Comment:
Yes, the naming is weird. I tested kind of every ImportAttribute field in an extra test method, since there were all the time issues and I wanted to work in small steps.
How could I improve it?